xla/service/gpu/transforms/windowed_einsum_handler.cc

```diff
@@ -282,7 +285,12 @@ absl::Status UpdateDotAndConsumerConfig(HloInstruction* dot,
   HloInstruction* updater = dot->users()[0];
   auto updater_gpu_config = updater->backend_config<gpu::GpuBackendConfig>();
   dot_gpu_config->set_operation_queue_id(stream_id);
-  updater_gpu_config->mutable_wait_on_operation_queues()->Add(stream_id);
+  const absl::flat_hash_set<int64_t> op_queue_id_set(
```

Comment:
Instead of converting to a set to check for stream_id, use `absl::c_linear_search(updater_gpu_config->wait_on_operation_queues(), stream_id)`